README.md

```diff
@@ -210,27 +120,18 @@ Example:
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@Olaoye](https://github.com/abayomiolaoye)
+- Twitter: [@Olaoye](https://twitter.com/olaoyeelijah)
+- LinkedIn: [Oluwatoyin Olaoye](https://linkedin.com/in/oluwatoyinolaoye)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **Use React for the client design**
+- [ ] **Implement some mmotions and transitions**
```

Comment:
- [ ] Please add the correct future features like `Validations and Model specs.` and `Processing data in models.`. 
You can get them all from the sneak peek lesson [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek_v1_1.md#:~:text=Project%202%3A%20Validations,Exercise%3A%20API%20documentation.).